tck/src/main/java/com/github/bsideup/liiklus/records/tests/SubscribeTest.java

```diff
@@ -144,4 +149,25 @@ default void testNullKey() throws Exception {
                     assertThat(it.getOffset()).isEqualTo(offsetInfo.getOffset());
                 });
     }
+
+    @Test
+    default void testValidTimestamp() throws Exception {
```

Comment:
To keep this PR more focused, I extracted this test and the fix into a separate PR with you as the author:
https://github.com/bsideup/liiklus/pull/177

I hope you don't mind :)